frontends/ios/BitBoxApp/BitBoxApp/BitBoxAppApp.swift

```diff
@@ -8,6 +8,7 @@
 import SwiftUI
 import Mobileserver
 import LocalAuthentication
+import UIKit
```

Comment:
What is this used for?

Reply:
This line somehow jumped from https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/2939 to here :sweat_smile: 

Removed